env/staging/rds/terragrunt.hcl

```
@@ -35,7 +35,7 @@ include {
 
 inputs = {
   eks_cluster_securitygroup = dependency.eks.outputs.eks-cluster-securitygroup
-  rds_instance_count        = 1
+  rds_instance_count        = 3
```

```suggestion
  rds_instance_count        = 2
```
This will be enough to perform tests, you'll have a writer and a reader instance with that. We'll minimize the cost.